modules/vault-elb/variables.tf

```diff
@@ -117,6 +117,12 @@ variable "health_check_timeout" {
 }
 
 variable "lb_tags" {
-  description = "Tags to be applied to the load balancer."
+  description = "Tags to be applied to the ELB."
+  default     = {}
+}
+
+variable "sg_tags" {
```

Comment:
`security_group_tags`?

Reply:
Internal testing, same as https://github.com/hashicorp/terraform-aws-vault/pull/83. I have two clusters (blue/green) per region (three regions) in two accounts (non-prod & prod).

I went with 'sg_tags' to match https://github.com/hashicorp/terraform-aws-vault/pull/81/files where I introduced 'lb_tags'. In retrospect, I wish I had used 'load_balancer_tags' in #81 so that this one could be 'security_group_tags' to match consul. At the time I did #81 I had not yet looked into the consul side. It would be a breaking change but I would *really* like to go with the longer names if agree.

Reply:
Yea, longer names will be more readable. Breaking change is OK.